101-bundle-json.md

```diff
@@ -743,14 +743,17 @@ Implementations MAY support user-defined additional actions as well. Such action
 {
   "actions": {
     "io.cnab.dry-run": {
+      "display": "Dry Run",
       "description": "prints what install would do with the given parameters values",
       "modifies": false,
       "stateless": true
     },
     "io.cnab.migrate": {
+      "display": "Migratte",
```

Comment:
I believe this should be `"Magritte"` 😉 

![art-magritte-listening-chamber-450x367px](https://user-images.githubusercontent.com/9322017/74486284-daf94500-4e79-11ea-9852-c603832fba96.jpg)